nicoRDL/javascript30-activity
# 05 dynamic flex 1.1

**Related Tickets**
JS30 - 05 Challenge

**Description**
implements and showcases a dynamic use of flexbox
expands and presents additional text when a panel is clicked

**Validation**
1. Open dynamicFlex.html
2. click on the panels to see transitions

**Risks**
1. certain errors may occur when using different browsers due to the flex property name. Current fix is done by selecting all property names with flex in them, which is limited by the flex changes happening when a panel opens.
2. Clicking on a new panel DOES NOT revert the originally clicked panel back. Code is designed to just listen for the click and transition the panel, not to revert back any other panel opened before.
